### ISSUE TITLE
Enum defaults are all invalid

### ISSUE THREAD

**Issue by VinSpee** [graphcool/console#761]
Hi!

I tried to add an Enum with a default today and no matter what, the default value was thowing an invalid error.

replication:

1. create a new Enum property
2. add some options
3. set a default
4. choose one from the list
5. save changes
6. see error

<img width="893" alt="screen shot 2017-03-18 at 3 18 15 pm" src="https://cloud.githubusercontent.com/assets/582828/24075348/c8e6d8f2-0bef-11e7-8116-4e613699ef5d.png">

I inspected the request and anticipate that it might be a string interpolation issue:

```json
{
  "query": "mutation UpdateFieldMutation($input_0:UpdateFieldInput!) {updateField(input:$input_0) {clientMutationId,...F1}} fragment F0 on Field {isUnique,id,typeIdentifier,isSystem,isRequired,name,isList,description,model {id},relatedModel {id,name},relation {id,name,description}} fragment F1 on UpdateFieldPayload {field {relatedModel {id,name,namePlural,_fields4tqA1b:fields(first:1000) {edges {node {typeIdentifier,name,id},cursor},pageInfo {hasNextPage,hasPreviousPage}}},reverseRelationField {id,isList,name},typeIdentifier,isList,isReadonly,isSystem,defaultValue,id,name,isRequired,enumValues,model {id,name},relationSide,relation {fieldOnLeftModel {id,name},fieldOnRightModel {id,name},id,name,description},isUnique,description,...F0}}",
  "variables": {
    "input_0": {
      "id": "cj0fmy1os0f730187de1sz7pv",
      "name": "role",
      "typeIdentifier": "Enum",
      "enumValues": [
        "ATTENDEE",
        "ORGANIZER",
        "SPEAKER",
        "SPONSOR",
        "VOLUNTEER"
      ],
      "isRequired": true,
      "isList": true,
      "isUnique": false,
      "defaultValue": "[\"ATTENDEE\"]",
      "clientMutationId": "e"
    }
  }
}
```

I'm assuming `defaultValue` should be `"ATTENDEE"` instead of `"[\"ATTENDEE\"]"`

**Comment by sorenbs** [comment]
Thanks for the detailed report - makes it much easier for us to investigate!

I have looked into this.

1) this only happens for list fields. If you uncheck the `store multiple values` box, setting a default enum value will work

2) this is a bug in the api - currently we don't support default values for enum list fields.

Please subscribe to https://github.com/graphcool/api-bugs/issues/20 for updates on this

**Comment by VinSpee** [comment]
what if I want to store multiple values that are any of the enum values? for example, a user could look like this:

```
roles: ["VOLUNTEER", "SPEAKER"]
```

**Comment by VinSpee** [comment]
I think i understand. The bug is on setting a default value when multiple values are possible on an `ENUM`?

**Comment by sorenbs** [comment]
That's exactly right! If you don't have a default value, this should work fine. We'll get this sorted in a few days :-)

**Comment by THook** [comment]
Not sure if it's exactly related but I can't manage to filter queries by values inside a list of ENUMS :

![capture d ecran 2017-03-28 a 13 09 45](https://cloud.githubusercontent.com/assets/6568065/24402243/f3274f64-13b7-11e7-9ad5-f32a11e28d0a.png)

"interestedBy" is a list of genders I am interested in. I can retrieve the values but the field is not listed in the filter docs.

That may be related because I added this field after I created the model, and created a random migration value to test things out!
